tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/ClientCredentialWithCertTest.cs

```diff
@@ -173,7 +173,9 @@ public async Task JsonWebTokenWithX509PublicCertSendCertificateOnBehalfOfTestAsy
                 Assert.IsNotNull(result.AccessToken);
 
                 appCacheAccess.AssertAccessCounts(0, 0);
-                userCacheAccess.AssertAccessCounts(1, 1);
+                userCacheAccess.AssertAccessCounts(0, 1);
+
+                harness.HttpManager.AddMockHandler(CreateTokenResponseHttpHandler(false));
```

Comment:
There are OBO tests that are commented. I think this is a good oportunity to bring them back to life.

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/bd8b7f693a01a058322198d4deb782e3eba831fe/tests/Microsoft.Identity.Test.Unit.net45/RequestsTests/OboFlowTests.cs#L17

Reply:
I think they were commented out when we moved to common core or smth. So they are pretty out of date possibly.